catalog/registry/catalog-registry-identificationplugin/pom.xml

```diff
@@ -59,20 +56,10 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>ddf.catalog.registry</groupId>
-            <artifactId>catalog-registry-common</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
         <dependency>
-            <groupId>ddf.catalog.registry</groupId>
-            <artifactId>catalog-registry-schema-bindings</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
             <groupId>ddf.platform</groupId>
             <artifactId>platform-parser-xml</artifactId>
```

Comment:
This is actually duplicate... fix while in there? Determine if scope should be test for the one left in.